tests/foreman/longrun/test_oscap.py

```diff
@@ -493,3 +493,131 @@ def test_positive_reporting_emails_of_oscap_reports():
+                'baseos': settings.repos.rhel8_os.baseos,
+                'appstream': settings.repos.rhel8_os.appstream,
+            }
+        )
+        vm.create_custom_repos(
+            sat_client=settings.repos['SATCLIENT_REPO']['RHEL8'],
+        )
+
+        result = vm.register(module_org, None, ak_name[distro], target_sat)
+        assert result.status == 0, f'Failed to register host: {result.stderr}'
+        proxy_id = target_sat.nailgun_smart_proxy.id
+        target_host = vm.nailgun_host
+        target_sat.api.AnsibleRoles().sync(
+            data={'proxy_id': proxy_id, 'role_names': [SELECTED_ROLE]}
+        )
+        role_id = (
+            target_sat.api.AnsibleRoles().search(query={'search': f'name={SELECTED_ROLE}'})[0].id
+        )
+        target_sat.api.Host(id=target_host.id).add_ansible_role(data={'ansible_role_id': role_id})
+        host_roles = target_host.list_ansible_roles()
+        assert host_roles[0]['name'] == SELECTED_ROLE
+        target_sat.cli_factory.make_scap_policy(
+            {
+                'scap-content-id': scap_id,
+                'hostgroups': hgrp_name,
+                'deploy-by': 'ansible',
+                'name': policy_name,
+                'period': OSCAP_PERIOD['weekly'].lower(),
+                'scap-content-profile-id': scap_profile_id,
+                'weekday': OSCAP_WEEKDAY['friday'].lower(),
+                'organizations': module_org.name,
+            }
+        )
+
+        vm.execute(f'curl -o {file_name} {download_url}')
+        target_sat.cli.Host.update(
+            {
+                'name': vm.hostname.lower(),
+                'lifecycle-environment': lifecycle_env.name,
+                'content-view': content_view.name,
+                'hostgroup': hgrp_name,
+                'openscap-proxy-id': default_proxy,
+                'organization': module_org.name,
+                'ansible-role-ids': role_id,
+            }
+        )
+
+        template_id = (
+            target_sat.api.JobTemplate()
+            .search(query={'search': 'name="Ansible Roles - Ansible Default"'})[0]
+            .id
+        )
+        job = target_sat.api.JobInvocation().run(
+            synchronous=False,
+            data={
+                'job_template_id': template_id,
+                'targeting_type': 'static_query',
+                'search_query': f'name = {vm.hostname}',
+            },
+        )
+        target_sat.wait_for_tasks(
+            f'resource_type = JobInvocation and resource_id = {job["id"]}',
+            poll_timeout=1000,
+        )
+        result = target_sat.api.JobInvocation(id=job['id']).read()
+        assert result.succeeded == 1
+        result = vm.run('cat /etc/foreman_scap_client/config.yaml | grep profile')
+        assert result.status == 0
+        # Runs the actual oscap scan on the vm/clients
+        result = vm.execute_foreman_scap_client()
+        assert f"WARNING: Using local file '/root/{file_name}'" in result
```

Comment:
Can you please mention TODO steps for future when the issue with SCAP run for policy will be fixed or add a `is_open('BZ:****')`  block to skip those steps and run `execute_foreman_scap_client()` if not?

Reply:
I will add steps